src/components/CallView/shared/EmptyCallView.vue

```diff
@@ -114,7 +122,11 @@ export default {
 			if (this.isConnecting) {
 				return t('spreed', 'Connecting …')
 			}
-			return t('spreed', 'Waiting for others to join the call …')
+			let callee = 'others'
+			if (this.isOneToOneConversation()) {
+				callee = this.getDisplayName()
+			}
+			return t('spreed', `Waiting for ${callee} to join the call …`)
```

Comment:
This breaks translations. So you need to make 2 translated strings:


```suggestion
			return t('spreed', `Waiting for others to join the call …`)
```